webapp/client/web_client.jsx

```diff
@@ -95,6 +95,19 @@ class WebClientClass extends Client {
             return success(res.body);
         });
     }
+
+    samlCertificateStatus(success, error) {
+        request.get(`${this.getAdminRoute()}/saml_cert_status`).
+        set(this.defaultHeaders).
+        type('application/json').
+        accept('application/json').
+        end((err, res) => {
+            if (err) {
+                return error(err);
+            }
+            return success(res.body);
+        });
+    }
```

Comment:
no, this one is only used from the system console, there's no need to add it in the driver